examples/010_harmonic_example.py

```diff
@@ -0,0 +1,241 @@
+"""
+.. _harmonic_example:
+
+Postprocess a harmonic analysis
+--------------------------------------
+
+.. note::
+
+    This example is work in progress. Please open a github issue if something is unclear.
+
+This example shows how to evaluate failure criteria for a harmonic simulation.
+It shows how to create a phase sweep to compute the maximum IRF in the frequency-phase
+space and shows how to identify the critical failure mode and the critical layer.
+
+"""
+
+
+# %%
+# Set up analysis
+# ~~~~~~~~~~~~~~~
+# Setting up the analysis consists of loading Ansys libraries, connecting to the
+# DPF server, and retrieving the example files.
+#
+# Load Ansys libraries.
```

Comment:
This description is somewhat misleading, since we're also loading `matplotlib`. Maybe we could either split the imports into two blocks, or change the description?